base-theme/assets/css/main.scss

```diff
@@ -41,6 +41,18 @@ a {
   }
 }
 
+p {
+  font-size: $normal-text-font-size;
+}
+
+li {
+  font-size: $normal-text-font-size;
+}
+
+table {
+  font-size: $normal-text-font-size;
+}
+
```

Comment:
Note to reviewer:
If there are other any other elements too which have the body font-size (i.e: `16px`), they can also be listed here.
 
Ideal case/solution  is to give this font-size to `body` element so it is reflected everywhere but it has already been given a font-size of `16px`, not sure why, and `p` element has been given a font-size of `18px`, again not sure why we have this difference at the first place so I have followed/extended `p` font-size (i.e: `18px`).